lib/mayaUsd/ufe/UsdUndoInsertChildCommand.cpp

```diff
@@ -132,8 +133,6 @@ UsdUndoInsertChildCommand::UsdUndoInsertChildCommand(
     // Apply restriction rules
     ufe::applyCommandRestriction(childPrim, "reparent");
     ufe::applyCommandRestriction(parentPrim, "reparent");
-
-    _childLayer = childPrim.GetStage()->GetEditTarget().GetLayer();
 }
```

Comment:
The old code assumed the original opinions about the prim that will be grouped were all in the current edit target. When the commands are being routed to other layers, that is not true. As a matter of fact, it was never true but just happened to work because we tend to use USD with a single layer.

The new code below correctly finds the source layer where the opinions actually are.